tracing.go

```diff
@@ -34,6 +34,11 @@ import (
 	"golang.org/x/net/context"
 )
 
+// TracerProvider returns an OpenTracing Tracer
+type TracerProvider interface {
```

Comment:
Is there any advantages to having this be public? While the other option would duplicate the interface, I wonder if it's better to keep the tchannel API smaller